src/components/TrackFilePicker.demo.js

```diff
@@ -0,0 +1,25 @@
+import Demo, {props as P} from 'react-demo'
+// See https://github.com/rpominov/react-demo for how to make a demo
+
+import React from "react";
+
+import TrackFilePicker from "./TrackFilePicker";
+
+export default (<Demo
+  props={{
+    fileOptions: P.json(["fileA", "fileB", "fileC", "anotherfile.vg"]),
+    fileSelect: P.string("fileA"),
+    pickerType: P.string("dropdown"),
```

Comment:
```suggestion
    pickerType: P.choices(["dropdown", "upload"]),
```

This will let the type value not have to go through unacceptable intermediate states like "dropdo" as the user is typing.